flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/api/internal/TableEnvironmentImpl.java

```diff
@@ -1176,6 +1178,31 @@ public TableResult executeInternal(Operation operation) {
                                         .getTableIdentifier()
                                         .asSerializableString()));
             }
+        } else if (operation instanceof ShowCreateViewOperation) {
+            ShowCreateViewOperation showCreateViewOperation = (ShowCreateViewOperation) operation;
+            Optional<CatalogManager.TableLookupResult> result =
+                    catalogManager.getTable(showCreateViewOperation.getViewIdentifier());
+            if (result.isPresent()) {
```

Comment:
We can move this check into the Optional, which will save us from one level of indentation:

```
final CatalogManager.TableLookupResult result = catalogManager
    .getTable(showCreateViewOperation.getViewIdentifier())
    .orElseThrow(() -> new ValidationException(…));

return TableResultImpl.builder()…
```

Reply:
OK. I'll update it.